django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -135,5 +139,34 @@ class DisbandTeamAPIView(APIView):
     )
     def delete(self, request, *args, **kwargs):
         team_name = kwargs["team_name"]
-        team_services.disband_team(user=request.user, team_name=team_name)
+        disband_team(user=request.user, team_name=team_name)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class UpdateTeamAPIView(APIView):
+    permission_classes = [IsAuthenticated]
+    serializer_class = CyberstormTeamUpdateSerializer
+    http_method_names = ["patch"]
+
+    @conditional_swagger_auto_schema(
+        operation_id="cyberstorm.team.update",
+        tags=["cyberstorm"],
+        request_body=CyberstormTeamUpdateSerializer,
+        responses={status.HTTP_200_OK: serializer_class},
+    )
+    def patch(self, request, *args, **kwargs):
+        team = get_object_or_404(
+            Team.objects.exclude(is_active=False), name=kwargs["team_name"]
```

Comment:
Rather than referring to `kwargs["team_name"]`, you could add it as a parameter on the function signature as `team_name: str` and use that directly